integreat_cms/cms/models/abstract_content_model.py

```diff
@@ -268,7 +268,15 @@ def __repr__(self):
         :rtype: str
         """
         class_name = type(self).__name__
-        return f"<{class_name} (id: {self.id}, region: {self.region.slug}, slug: {self.best_translation.slug})>"
+        # Only log the slug if doing so does not require a database query
```

Comment:
Yeah, this is probably not necessary. I added this, because the `PageQuerySet.cache_tree` method logs all pages, where this method requires a databse call for every page. I did not keep in mind that this would only affect debug mode.